docs/dev/README.md

```diff
@@ -1,35 +1,52 @@
 
-[Custom Checks][1] are great for occasional reporting, or in cases where the data source is either unique or very limited. For more general use-cases - such as application frameworks, open source projects, or commonly-used software - it makes sense to write an integration.
+The [Datadog Marketplace][4], on the other hand, is a commercial platform for Technology Partners to _sell_ a variety of offerings, including integrations, custom dashboard widgets, software subscriptions/licenses, and professional services to Datadog customers.
 
-Metrics reported from accepted integrations are not counted as custom metrics, and therefore don't impact your custom metric allocation. (Integrations that emit potentially unlimited metrics may still be considered custom.) Ensuring native support for Datadog reduces friction to adoption, and incentivizes people to use your product, service, or project. Also, being featured within the Datadog ecosystem is a great avenue for added visibility.
+## Datadog Integrations
 
-### What's the process?
+### [API-based Integration][1]
+API-based integrations can enrich a customer's Datadog environment by submitting data from external technologies via the Datadog API. Customers authorize API integrations access to their accounts through OAuth, and partners write and host the implementation code that makes up the integration. API integrations work well for partners building a connector between Datadog and another SaaS platform.
 
-The initial goal is to generate some code that collects the desired metrics in a reliable way, and to ensure that the general integration framework is in place. Start by writing the basic functionality as a custom Check, then fill in the framework details from [Create a New Integration][2].
+### [Agent-based Integration][2]
+Agent-based integrations use the Datadog Agent to submit data via checks written by the partner. The implementation code for these integrations is hosted by Datadog. Agent integrations are best suited for collecting data from custom systems or applications. Writing an agent integration requires you to publish and deploy your solution as a Python wheel (.whl).
+
+## Why create an integration?
 
-Next, open a pull request against the [integrations-extras repository][3]. This signals to Datadog that you're ready to start reviewing code together. Don't worry if you have questions about tests, Datadog internals, or other topics - the integrations team is ready to help, and the pull request is a good place to go over those concerns. Be sure to take advantage of the [Community Office Hours][4] as well!
+Out-of-the-box Metrics - Metrics reported from official Datadog integrations (unless the integration is sending in potentially unlimited metrics) are not counted as custom metrics, and therefore won't impact a customer's custom metric allocation.
```

Comment:
Should we bold the first reason for readability? E.g., "**Adoption** - Ensuring..."

I also think Discoverability should go first. 

These reasons could be fleshed out a bit more. For example, I don't see anything about the main benefits of observability, correlation, resolving issue for users, etc.

Reply:
My latest commit didn't sync up to staging yet, but i bolded those terms :) 

@emarsha94 for the comment on fleshing out the reasons